packages/fxa-auth-server/lib/routes/utils/third-party-events.ts

```diff
@@ -132,21 +149,28 @@ export const appleEventHandlers = {
  * @param eventDetails
  * @param log
  * @param db
+ * @param statsd
  * @returns Promise<void>)
  */
 async function handleAppleConsentRevokedEvent(
   eventDetails: AppleSETEvent,
   log: any,
-  db: any
+  db: any,
+  statsd: StatsD
 ) {
-  const sub = eventDetails.sub;
-  const account = await getAccountFromSub(sub, db, 'apple', log);
-
-  // We have a guard that account exists because it is possible that it was
-  // removed in another security event
-  if (account) {
-    await revokeThirdPartySessions(account.uid, 'apple', log, db);
-    await db.deleteLinkedAccount(account.uid, 'apple');
+  try {
+    const sub = eventDetails.sub;
+    const account = await getAccountFromSub(sub, db, 'apple', log, statsd);
+
+    // We have a guard that account exists because it is possible that it was
+    // removed in another event
+    if (account) {
+      await revokeThirdPartySessions(account.uid, 'apple', log, db, statsd);
+      await db.deleteLinkedAccount(account.uid, 'apple');
+    }
+  } catch (error) {
+    statsd.increment('handleAppleConsentRevokedEvent.error');
```

Comment:
It might be nice to what part of this it failed on, `getAccountFromSub`, `revokeThirdPartySessions` or `db.deleteLinkedAccount`. We could capture the sentry error directly, e.g. `Sentry.captureException(error)`